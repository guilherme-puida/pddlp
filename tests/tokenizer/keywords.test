decrease
either
exists
end
forall
not
imply
preference
total-time
