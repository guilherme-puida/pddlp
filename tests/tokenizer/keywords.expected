[01:01] PDDLP_TOKEN_ALL all
[02:01] PDDLP_TOKEN_ALWAYS always
[03:01] PDDLP_TOKEN_ALWAYS_WITHIN always-within
[04:01] PDDLP_TOKEN_AND and
[05:01] PDDLP_TOKEN_ASSIGN assign
[06:01] PDDLP_TOKEN_DECREASE decrease
[07:01] PDDLP_TOKEN_DEFINE define
[08:01] PDDLP_TOKEN_DOMAIN domain
[09:01] PDDLP_TOKEN_EITHER either
[10:01] PDDLP_TOKEN_END end
[11:01] PDDLP_TOKEN_EXISTS exists
[12:01] PDDLP_TOKEN_FORALL forall
[13:01] PDDLP_TOKEN_HOLD_AFTER hold-after
[14:01] PDDLP_TOKEN_HOLD_DURING hold-during
[15:01] PDDLP_TOKEN_IMPLY imply
[16:01] PDDLP_TOKEN_INCREASE increase
[17:01] PDDLP_TOKEN_IS_VIOLATED is-violated
[18:01] PDDLP_TOKEN_MAXIMIZE maximize
[19:01] PDDLP_TOKEN_MINIMIZE minimize
[20:01] PDDLP_TOKEN_NOT not
[21:01] PDDLP_TOKEN_OBJECT object
[22:01] PDDLP_TOKEN_OR or
[23:01] PDDLP_TOKEN_OVER over
[24:01] PDDLP_TOKEN_PREFERENCE preference
[25:01] PDDLP_TOKEN_PROBLEM problem
[26:01] PDDLP_TOKEN_SCALE_UP scale-up
[27:01] PDDLP_TOKEN_START start
[28:01] PDDLP_TOKEN_TOTAL_TIME total-time
[29:01] PDDLP_TOKEN_UNDEFINED undefined
[30:01] PDDLP_TOKEN_WHEN when
[31:01] PDDLP_TOKEN_WITHIN within
EOF